src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFANewModifyPage.java

```diff
@@ -108,33 +106,36 @@ public void setIndustryList(final String industry) {
 
     /**
      * Method to set annualRevenue attribute.
+     *
      * @param annualRevenue the value.
      */
     public void setAnnualRevenueTextField(final String annualRevenue) {
         this.annualRevenueTextField.sendKeys(annualRevenue);
     }
 
 
-
     /**
-     * Method to click button.
+     * @param values account fields.
+     * @return map of account fields.
      */
-    public void clickSaveNewAccountButton() {
-        getSaveNewAccountButton().click();
-    }
+    public Map<Inputs, Value> getStrategyStepMap(final Map<Inputs, String> values) {
+        Map<Inputs, Value> strategyMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)